### PR TITLE
New guide: how to query upstream CL repo

### PR DESCRIPTION
**Pull request by MCamp859**
Addresses Issue #527  
Added to Guides > Maintenance 
@bktan8 @mvincerx  Please have a look. 
Signed-off-by: MCamp859 <maryx.camp@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/maintenance/query-upstream.rst [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+#.	Initialize the RPM database with the command:
+
+	.. code-block:: bash
+
+		sudo rpm --initdb
+
+
+
+DNF command usage
```
Comment (bktan8) [inline source/clear-linux/guides/maintenance/query-upstream.rst]: @MCamp859 - can you "examples" to the end of "DNF command usage"?

**Comment by MCamp859** [comment]
@bktan8  Can you please review/approve asap? We're trying to get this guide added to the "ready for translation" group.

**Comment by MCamp859** [comment]
@mvincerx   OK to merge.
@krisdale   OK to put in translation group.

**Comment by mvincerx** [comment]
Hold on any new commits. I need to resolve a couple minor merge conflicts on this branch.

**Comment by mvincerx** [comment]
Resolved conflicts. Clean build. LGTM.

**Pull request merged**